xml/preinstall-configure-deployer-repos.xml

```diff
@@ -31,8 +31,7 @@
   patches for the products and extensions. To make them available for
   &productname; they need to be mirrored from the &scc;. Since their content is
   regularly updated, they must be kept in synchronization with &scc;. For
-  these purposes, &suse; provides either the &smtool; (&smt;) or the
-  &susemgr;. See <xref linkend="sec.depl.adm_conf.repos.scc" /> for setup instructions.
+  these purposes, &suse; provides the &smtool; (&smt;). See <xref linkend="sec.depl.adm_conf.repos.scc" /> for setup instructions.
```

Comment:
Just a comment, should this be wrapped to a text limit? I can't see the end of the sentence.

Reply:
Ah I see it, it's just the end of the word "instruction"

Reply:
changed for the bettah.